### DISCUSSION TITLE
When creating a M2A relationship, can I select a non-PK column in the "Many" end?

### DISCUSSION THREAD

**Discussion by powerseed** [directus/directus#10356] [Archive]
The screenshot below is the UI when creating a M2A relationship.

![image](https://user-images.githubusercontent.com/42278237/144960194-06cec6de-3562-4860-89ba-ccc1324b8036.png)

As annotated above, for the table in the "Many" end, it is always the primary key being selected.

I am wondering if I can select anyother column?

Thanks!

**Comment by joselcvarela** [comment]
Hello @powerseed 
I believe that is currently not supported, since we are only relying on primary key.
But, can you describe a use case where you would need this?
Keep in mind that primary key is always indexed on database, so if we support such feature we would need to add indexes for those fields.

**Comment by rijkvanzanten** [comment]
No, you can't currently change what related column is used as the key in the foreign key constraint 👍🏻
